vignettes/tables.Rmd

```diff
@@ -48,7 +48,7 @@ genes<-data.frame(
 	Strand=c('+', '+', '-', '-', '+', '+'),
 	TranscriptionFactor=c(F,F,F,F,T,F))
 genesFile<-tempfile()
-write.csv(genes, file=genesFile, row.names=FALSE)
+saveToCsv(genes, genesFile)
 ```
```

Comment:
look into whether it's possible to change the Table() constructor to accept an R data frame rather than a file path (and to write the file, etc. 'under the hood').